MAUI/DataForm/localization.md

```diff
@@ -0,0 +1,182 @@
+
+## Setting CurrentUICulture to the application
+
+Application culture can be changed by setting `CurrentUICulture` in `App.xaml.cs` file.
+
+{% tabs %}
+{% highlight C# tabtitle="App.xaml.cs" %}
+
+using System.Globalization;
+
+public partial class App : Application
+{
+	public App()
+	{
+		InitializeComponent();
+
+		MainPage = new AppShell();
+        var culture = new CultureInfo("fr");
+        CultureInfo.CurrentUICulture = culture;
+    }
+}
+
+{% endhighlight %}
+{% endtabs %}
+
+N>
+The required `resx` files with `Build Action` as `EmbeddedResource` (File name should contain culture code) into the `Resources` folder.
+
+## Add Localization resx file in sample level
+
+To localize the `DataForm` based on `CurrentUICulture` using `resource` files, follow the below steps.
+
+1. Right-click on the `Resources` folder in the application.
+
+    ![addition-of-default-resource-file-of-maui-dataform-into-resources-folder](images/localization/addition-of-default-resource-file-of-maui-dataform-into-resources-folder.png)
+
+2. Select `Add` and then `NewItem`.
+3. In `Add New Item` wizard, select the `Resource File` option and name the filename as `DataFormLocalization.<culture name>.resx.` For example, give the name `DataFormLocalization.fr.resx` for French culture.
+
+    ![shows-the-name-of-resource-file-to-be-added-for-maui-dataform](images/localization/shows-the-name-of-resource-file-to-be-added-for-maui-dataform.png)
+
+4. The culture name indicates the name of the language and country.
+
+5. Now, select `Add` option to add the resource file in **Resources** folder.
+
+    ![shows-the-added-resource-file-for-french-language-in-maui-dataform](images/localization/shows-the-added-resource-file-for-french-language-in-maui-dataform.png)
+
+6. Add the Name/Value pair in Resource Designer of `DataFormLocalization.fr.resx` file and change its corresponding value to corresponding culture.
+ 
+    ![shows-the-added-resource-file-name-value-pair-in-the-resource-designer-in-maui-dataform](images/localization/shows-the-added-resource-file-name-value-pair-in-the-resource-designer-in-maui-dataform.png)
+
+## Localizing data form display values
+
+Here, the display attributes or data form item display values get localized based on culture from Localization Resource File (.Resx).
+
+#### Using attribute
+
+`ResourceType` property of [Display](https://learn.microsoft.com/en-us/dotnet/api/system.componentmodel.dataannotations.displayattribute?view=net-7.0) attribute specifies the Resources File (.Resx) which is used to localize the values of `Name`, `ShortName`, `GroupName`, `Prompt` and `ItemsSource`.
```

Comment:
`ResourceType` property of the [Display](https://learn.microsoft.com/en-us/dotnet/api/system.componentmodel.dataannotations.displayattribute?view=net-7.0) attribute specifies the Resources File (.Resx), which is used to localize the values of `Name,` `ShortName,` `GroupName,` `Prompt` and `ItemsSource.`